cli.js

```diff
@@ -56,8 +79,43 @@ function run(file) {
 			return;
 		}
 
-		require(file);
+		var options = {
+			env: assign({}, process.env, {AVA_FORK: 1}),
+			silent: true
+		};
+
+		files++;
+
+		var babel = join(__dirname, 'babel.js');
+
+		var ps = fork(babel, [file], options);
+
+		ps.stdout.on('data', test);
+		ps.stderr.on('data', test);
```

Comment:
I was thinking that only the CLI would print to stdout. The forked processes would message status updates through the message channel. That way the parallelism is contained to the API and the CLI only calls the API normally.